### PR TITLE
[loss] add sphereface2 loss function

### PR DESCRIPTION
**Pull request by Hunterhuan**
image

### CODE REVIEW

**Review by czy97 — APPROVED** [review]

**Comment by kafan1986** [comment]
@Hunterhuan any pre trained model available for testing?

**Comment by Hunterhuan** [comment]
> @Hunterhuan any pre trained model available for testing?

We only provide pre-trained models with basic setup. For sphereface2, you can train it by yourself. Just change the 'arc_margin' to 'sphereface2' in the config. Thanks.
